src/masternodes/rpc_masternodes.cpp

```diff
@@ -165,7 +174,12 @@ UniValue createmasternode(const JSONRPCRequest& request)
         throw JSONRPCError(RPC_INVALID_PARAMETER, strprintf("Address (%s) is not owned by the wallet", EncodeDestination(ownerDest)));
     }
 
+    if (rewardDest.index() != 1 && rewardDest.index() != 4) {
```

Comment:
Remove this check and let it happen on the consensus side. Reward will include type 2 as well on the next hard fork due to PR 1664.